.yarnrc

```diff
@@ -0,0 +1,2 @@
+
+workspaces-experimental true
```

Comment:
Could you remove this file? I think with `yarn@1.x` this is no longer required.

Reply:
removed